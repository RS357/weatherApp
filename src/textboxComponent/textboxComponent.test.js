import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import TextboxComponent from './textboxComponent.js';



describe('TextboxComponent should', () => {
    it ('exist', () => {
        expect(TextboxComponent).toBeDefined()   
    });

    it('have a consistent structure', () => {
        const wrapper = mount(<TextboxComponent />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})