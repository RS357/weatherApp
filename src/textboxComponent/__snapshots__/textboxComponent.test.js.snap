// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextboxComponent should have a consistent structure 1`] = `
<Textbox>
  <input
    placeholder="city"
    type="text"
  />
</Textbox>
`;
